src/world/World.ts

```diff
@@ -443,6 +443,17 @@ export class World extends EventTarget {
     this.contactMaterialTable.set(cmat.materials[0].id, cmat.materials[1].id, cmat)
   }
 
+  /**
+   * Removes a contact material from the World.
+   */
+  removeContactMaterial(cmat: ContactMaterial): void {
+    const idx = this.contactmaterials.indexOf(cmat)
+    if (idx !== -1) {
+      this.contactmaterials.splice(idx, 1)
+    }
+    this.contactMaterialTable.delete(cmat.materials[0].id, cmat.materials[1].id)
```

Comment:
```suggestion
    if (idx === -1) {
      return
    }
    
    this.contactmaterials.splice(idx, 1)
    this.contactMaterialTable.delete(cmat.materials[0].id, cmat.materials[1].id)
```

why is `contactMaterialTable.delete` not under the check?

Reply:
I think I copied from `removeConstraint` method.

No objections to the suggested change.

Pushed update.